README.md

```diff
@@ -93,40 +93,90 @@ Refer to [AST Serialization Media Types](https://github.com/apiaryio/api-bluepri
 ## Build
 1. Clone the repo + fetch the submodules:
 
-	```sh
-	$ git clone --recursive git://github.com/apiaryio/drafter.git
-	$ cd drafter
-	```
+    ```sh
+    $ git clone --recursive git://github.com/apiaryio/drafter.git
+    $ cd drafter
+    ```
 
 2. Build & test Drafter:
 
-	```sh
-	$ ./configure
-	$ make test
-	```
+    ```sh
+    $ ./configure
+    $ make test
+    ```
 
-	To include integration tests (using Cucumber) use the `--include-integration-tests` flag: 
+    To include integration tests (using Cucumber) use the `--include-integration-tests` flag:
+
+    ```sh
+    $ ./configure --include-integration-tests
+    $ make test
+    ```
 
-	```sh
-	$ ./configure --include-integration-tests
-	$ make test
-	```
-	
 We love **Windows** too! Please refer to [Building on Windows](https://github.com/apiaryio/drafter/wiki/Building-on-Windows).
-		
+
 ### Drafter command line tool
 1. Build `drafter`:
-	
-	```sh
-	$ make drafter
-	```
+
+    ```sh
+    $ make drafter
+    ```
 
 2. Install & use `drafter`:
 
-	```sh
-	$ sudo make install
-	$ drafter --help
-	```
+    ```sh
+    $ sudo make install
+    $ drafter --help
+    ```
+
+## jsdrafter.js
+
+*Note: jsdrafter.js is work in progress, use only for experimental purposes*
+
+`jsdrafter.js` is a pure JavaScript version of the `drafter` library. It
+exposes 2 functions `string parseAPIBlueprint(string)` and `int
+validateAPIBlueprint(string)` which serve for getting a parser output
+in JSON format and to just validate the API Blueprint respectively.
+It is build from the C++ source using
+[emscripten](http://kripken.github.io/emscripten-site/).
+
+
+### Build jsdrafter.js
+
+*Unfortunately it works only on *nix environment at the moment.*
```

Comment:
Clarify that it is only **building** that requires *nix. Using the library works everywhere.